File: Assets/Game/Hoftix/Script/GenerateScripts/UIModules/BagUIPanelView.cs
using ZJYFrameWork.UISerializable.UIInitView;
using UnityEngine;
using UnityEngine.UI;

namespace ZJYFrameWork.UISerializable
{
    public class BagUIPanelView:UIViewInterface
    {
        public UnityEngine.GameObject EquipAndItemPanel=null;
		


        public void Init(UIView _view)
        {
            EquipAndItemPanel=_view.GetObjType<UnityEngine.GameObject>("EquipAndItemPanel");
			
        }
    }
}

File: DemoMMOUnity/Lua/application/app/ui/generateScripts/UIModules/BagUIPanelView.lua
---@class BagUIPanelView
BagUIPanelView = class("BagUIPanelView")
function BagUIPanelView:Init(view)
	self._UIView = view:GetComponent("UIView")
	self.equipmentItem_UISerializableKeyObject = self._UIView:GetObjType("equipmentItem_UISerializableKeyObject") or ZJYFrameWork.UISerializable.UISerializableKeyObject
	self.topSelectTipsShowText = self._UIView:GetObjType("topSelectTipsShowText") or UnityEngine.UI.Text
	self.CloseBtn = self._UIView:GetObjType("CloseBtn") or UnityEngine.UI.Button
	self.equipmentButton = self._UIView:GetObjType("equipmentButton") or UnityEngine.UI.Button
	self.equipmentButtons_UISerializableKeyObject = self._UIView:GetObjType("equipmentButtons_UISerializableKeyObject") or ZJYFrameWork.UISerializable.UISerializableKeyObject
	self.WeaponInDetailItem_UISerializableKeyObject = self._UIView:GetObjType("WeaponInDetailItem_UISerializableKeyObject") or ZJYFrameWork.UISerializable.UISerializableKeyObject
	self.weaponGrid = self._UIView:GetObjType("weaponGrid") or UnityEngine.GameObject
end

